src/uucore/src/lib/features/format/spec.rs

```diff
@@ -349,7 +349,7 @@ impl Spec {
             Self::EscapedString => {
                 let s = args.get_str();
                 let mut parsed = Vec::new();
-                for c in parse_escape_only(s.as_bytes(), OctalParsing::default()) {
+                for c in parse_escape_only(s.as_bytes(), OctalParsing::ThreeDigits) {
```

Comment:
Just to make sure, this doesn't break stuff like this right?

```
env printf "%b" "\\05llo" | hexdump -C
00000000  05 6c 6c 6f                                       |.llo|
env printf "%b" "\\5llo" | hexdump -C
00000000  05 6c 6c 6f                                       |.llo|
```

Reply:
It doesn't break it (this is with GNU coreutils 9.6 in /bin):
![Screenshot_20250403_062101](https://github.com/user-attachments/assets/2d809e94-d489-4161-91a4-48a089e9a139)

I added some tests to show that octal characters are still accepted at different lengths (1-3 numbers normally and 1-4 with leading zero for %b)

Reply:
Thanks!

Reply:
please avoid screenshot.
They are terrible for accessibility and search

Reply:
Thank you for the reminder, I replaced the screenshot